portfolio/src/main/webapp/script.js

```diff
@@ -40,26 +37,48 @@ function addHobby() {
 function addProjectDescription(project) {
   
   const details = {
-    nlp: 'Building language models',
-    ta: 'Held weekly office hours and review sessions for introductory Data Structure and Algorithms class',
-    helmet: 'Integrated speech command to a \"smart\" bike helmet built by a team of 9 friends',
+    Nlp: 'Building language models',
```

Comment:
These keys seem like they represent constant keys. So I think we can use all uppercase letters.

ex:

const details = {
  NLP: ...
  TA: ...
}

Reference:
https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
Done